Makefile

```diff
@@ -12,33 +12,20 @@ all_artifacts:
 	open ./Products
 	echo "🏁 Ended at $$(date +%T)"
 
-frameworks: clean
-	echo "👉 Creating dynamic libraries. Will take a while... Logs available: DerivedData/fastlane.log"
-	bundle exec fastlane build_xcframeworks > DerivedData/fastlane.log
-	echo "👉 Creating compressed archives"
-	cp ./LICENSE ./Products/LICENSE
-	make zip_artifacts name="StreamChat-All" pattern=./*.xcframework
-	make zip_artifacts name="StreamChat" pattern="./StreamChat.xcframework ./LICENSE"
-	make zip_artifacts name="StreamChatUI" pattern="./StreamChatUI.xcframework ./LICENSE"
-	make swiftpm_checksum
-	make clean
+frameworks:
+	bundle exec fastlane clean_products
+	echo "👉 Creating dynamic libraries. Will take a while... Logs available: fastlane/fastlane.log"
+	bundle exec fastlane build_xcframeworks > fastlane/fastlane.log
+	bundle exec fastlane compress_frameworks
+	bundle exec fastlane clean_products
 
-static_libraries: clean
+static_libraries:
+	bundle exec fastlane clean_products
 	echo "👉 Creating static libraries"
 	./Scripts/buildStaticLibraries.sh
 	echo "👉 Creating compressed archive"
 	make zip_artifacts name="StreamChat-All-Static" pattern='./*.xcframework ./*.bundle'
-	make clean
-
-clean:
-	echo "♻️  Cleaning ./Products & ./DerivedData"
```

Comment:
Was nice to have visible logs here

Reply:
But it still prints the logs about it 🤔
![Screenshot 2023-02-20 at 12 18 49 PM](https://user-images.githubusercontent.com/20794991/220105150-1bf85d05-4633-4fb0-99f2-a27292accd33.png)

Reply:
Returned the logs 👍